src/SourceBuild/content/repo-projects/dotnet.proj

```diff
@@ -23,23 +23,24 @@
     <RepositoryReference Include="razor" />
     <RepositoryReference Include="xliff-tasks" />
     <RepositoryReference Include="cecil" />
+    <RepositoryReference Include="symreader" />
+    <RepositoryReference Include="source-build-externals" />
     <RepositoryReference Include="runtime" />
     <RepositoryReference Include="roslyn" />
-    <RepositoryReference Include="source-build-externals" />
-    <RepositoryReference Include="symreader" />
     <RepositoryReference Include="xdt" />
     <RepositoryReference Include="msbuild" />
     <RepositoryReference Include="roslyn-analyzers" />
     <RepositoryReference Include="aspnetcore" />
     <RepositoryReference Include="deployment-tools" />
     <!-- TODO: Reenable once format targets net8.0 (https://github.com/dotnet/format/issues/1802) -->
     <!-- <RepositoryReference Include="format" /> -->
-    <RepositoryReference Include="templating" />
+    <RepositoryReference Include="format" />
```

Comment:
Doesn't this need to be commented out?

Reply:
That was a result of a bad merge on my part.  The line should actually be remove as there is already a commented out version.  Thanks for the catch.